index.js

```diff
@@ -38,8 +47,53 @@ app.post('/posts/:id/comments', (req, res) => {
 });
```

Comment:
web forms doesnt support PUT or DELETE request, thus this middleware turns a POST request to PUT or DELETE if the form has an "_method" field with the respective value